chapters/VK_EXT_present_timing/PresentTimeInfo.adoc

```diff
@@ -0,0 +1,180 @@
+// Copyright (c) 2014-2023 Khronos Group.
+//
+// SPDX-License-Identifier: CC-BY-4.0
+
+[open,refpage='VkPresentTimingsInfoEXT',desc='Array of VkPresentTimingInfoEXT to chain with VkPresentInfoKHR',type='structs']
+--
+
+When the <<features-presentAtAbsoluteTime, pname:presentAtAbsoluteTime>> or
+<<features-presentAtRelativeTime, pname:presentAtRelativeTime>> feature is
+enabled, an application can: instruct the presentation engine that an image
+must: not be displayed earlier than a specified time, or for a minimum
+duration, by including the sname:VkPresentTimingsInfoEXT structure in the
+pname:pNext chain of the slink:VkPresentInfoKHR structure.
+
+The sname:VkPresentTimingsInfoEXT structure is defined as:
+
+include::{generated}/api/structs/VkPresentTimingsInfoEXT.adoc[]
+
+  * pname:sType is a elink:VkStructureType value identifying this structure.
+  * pname:pNext is `NULL` or a pointer to a structure extending this
+    structure.
+  * pname:swapchainCount is the number of swapchains being presented to by
+    this command.
+  * pname:pTimingInfos is `NULL` or a pointer to an array of
+    sname:VkPresentTimingInfoEXT elements with pname:swapchainCount entries.
+    If not `NULL`, each element of pname:pTimingInfos contains timing
+    information for the presentation of the image corresponding to the entry
+    in the sname:VkPresentInfoKHR::pname:pImageIndices array.
+
+The semantics of target presentation time or duration vary depending on the presentation mode:
+
+  * ename:VK_PRESENT_MODE_FIFO_KHR.
+    Tearing cannot: be observed.
+    The first queued image is dequeued and presented if
+    both: 1) its associated wait semaphore(s) have signaled, and 2)
+    its target present time is less-than or equal-to the current time.
+  * ename:VK_PRESENT_MODE_FIFO_RELAXED_KHR.
+    For images that are presented in time to be displayed at the next
+    vertical blanking period, the semantics are identical as for
+    ename:VK_PRESENT_MODE_FIFO_KHR.
+    For images that are presented late, and are displayed after the start of
+    the vertical blanking period (i.e. with tearing), the values of
+    sname:VkPastPresentationTimingEXT may: be treated as if the image was
+    displayed at the start of the vertical blanking period, or may: be
+    treated the same as for ename:VK_PRESENT_MODE_IMMEDIATE_KHR.
```

Comment:
That seems like it implies having `VK_PRESENT_STAGE_IMAGE_FIRST_PIXEL_VISIBLE_BIT` timing information and a vblank time to compare it to. If the implementation has access to those, it should just expose them: otherwise, the next question is "how late was the image". There's nothing in the spec currently regarding vblank specifically though, and I'm not sure we'd want to go down that way.